packages/api-kit/tests/.env-sample

```diff
@@ -0,0 +1,2 @@
+# Transaction Service API Key
+TX_SERVICE_API_KEY=
```

Comment:
This file should be at `api-kit` top level, not inside `tests`folder

Reply:
done